lib/net/networking_engine.js

```diff
@@ -429,6 +431,7 @@ shaka.net.NetworkingEngine = class extends shaka.util.FakeEventTarget {
               shaka.util.Error.Code.UNSUPPORTED_SCHEME,
               uri));
     }
+    const progressSupport = object ? object.progressSupport : false;
```

Comment:
I think this can be object.progressSupport.  By this point in the code, you know object is non-null, right?  Or does the compiler not understand that detail?

Reply:
Sometimes I want to be too purist, sorry, now I change it.